readthedocs/templates/core/project_bar_base.html

```diff
@@ -41,7 +41,7 @@ <h1>
 
         <li class="{{ downloads_active }}"><a href="{% url "project_downloads" project.slug %}" rel="nofollow,noindex">{% trans "Downloads" %}</a></li>
 
-        <li class="{{ search_active }}"><a href="{% url "elastic_project_search" project.slug %}" rel="nofollow,noindex">{% trans "Search" %}</a></li>
+        <li class="{{ search_active }}"><a href="{% url "search" %}?q=project:{{ project.slug|urlencode }}%20" rel="nofollow,noindex">{% trans "Search" %}</a></li>
```

Comment:
This feels like a weird UX.. I think we probably just remove this from the menu until we can make a nicer way to make a global search bar work well? We can definitely have "search this project" in some other UI element though, but in the menu it feels odd to end up outside the project context.

Reply:
Would this be like a link on the detail view? I kind of like it having a direct way to search the project. Do we have a way to know how many people click/use this in GA? :thinking:

Reply:
We should definitely have a nice way to search the project, but it feels odd as a UI element. We should find another way to expose it I think.

Reply:
@agjohnson any ideas/suggestions about where to put this `search this project` link?

Reply:
I was looking for a solution like this for the contextualized 404 pages :tada: 

So for now, I can refer to `{% url "elastic_project_search" project.slug %}` and then update it once the API is out?

Reply:
yeah, that should be fine